docs/resources/widgets/widgets.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+---
+# 
+## What's a widget?
+
+In FlutterFlow, a widget is like a building block used to create the user
+interface of your app. You can think of it as a UI element that you can use and
+customize to build different parts of your app's screen, such as buttons, text
+fields, images, and more.
+
+Widgets can be combined and nested to create complex layouts and interactive
+elements, making it easier to design your app visually. FlutterFlow provides a
+drag-and-drop interface, so you can place these widgets on your app's canvas and
+adjust their properties through the interactive interface, without having to
+write a lot of code.
+
+### Everything's a widget
+
+FlutterFlow is built with Flutter, Google's popular framework for building
+cross-platform apps from a single codebase. In Flutter, _everything is a widget_
+—every part of the app's UI is built using widgets that handle layout, styling,
+and interaction. FlutterFlow leverages a similar concept by providing a visual
+editor where you can create and configure these widgets.
+
+![everything-widget.png](imgs%2Feverything-widget.png)
+
+## Widget categories
+
+In FlutterFlow, we have the following categories of widgets:
+
+- Layout Elements
+- Base Elements
+- Page Elements
+- Form Elements
+
+### Layout Elements
```

Comment:
For each of these categories, it would be nice to have a table - with an example of the widget in use / image of the example

Reply:
For example, one row would be for "column widget" - we would have the name / description in one column, and make some visual diagram of a row being used on a page. 

I think we just need some more visuals here, feels too text heavy